### PR TITLE
tldr: add --language example

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by navarroaxel** [comment]
Comes from this discussion https://github.com/tldr-pages/tldr/discussions/8270#discussioncomment-3289159

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Hi, I don't think we can merge this PR since the tldr.md page can only have command example which are a `MUST` for all clients. See [here](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#arguments).

**Comment by kbdharun** [comment]
Thanks for your replies. I will close the pull request due to the above mentioned reasons.
